packages/cli/src/Version.ts

```diff
@@ -96,6 +97,7 @@ export class Version implements Command {
       ['Current platform', platform],
 
       ...enginesRows,
+      ['Format WASM', `@prisma/prisma-fmt-wasm ${wasm.prismaFmt.version()}`],
```

Comment:
Offical capitalization:
```suggestion
      ['Format Wasm', `@prisma/prisma-fmt-wasm ${wasm.prismaFmt.version()}`],
```

Reply:
Fixed this in a commit that also updates the snapshots in `Version.test.ts`